tests/payment/001/c/cpayment.spec.js

```diff
@@ -44,6 +44,9 @@ const Nightmare = require( "nightmare" ),
         });
 
         it ("should submit with all forms filled for 'try now' button", function(done) {
-            myModule.formABCD_input(browser, done);
+            console.log("abstracted c");
+            var selectors = {"address": "#address1", "city": "#city", "state": "#state", "checkbox": "input[type='checkbox']", "name": "input[name='name_on_card']", "cardnum": "input[name='card_number']", "cvv": "input[name='cvv']", "zipcode": "input[name='zipcode']", "exp": "input[name='expirationMY']"};
```

Comment:
Since all payment tests use similar selectors, we could extract the selectors to a file and then all tests can include it.